src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -60,3 +57,41 @@ $content-width: $gene_image_width;
   justify-content: center;
   margin-top: 40px;
 }
+
+.xrefWithChevron {
+  display: flex;
+}
+
+.xrefGroupChevron {
+  text-align: right;
+  padding-right: 2px;
+  cursor: pointer;
+  font-weight: $normal;
+  min-width: 50px;
+}
+
+.chevron {
+  margin-left: 6px;
+  margin-bottom: -1px;
+  height: 12px;
+  width: 12px;
+}
+
+.chevronUp {
+  transform: rotate(0deg);
+}
+
+.chevronDown {
+  transform: rotate(-180deg);
+}
+
+.xrefsWrapper {
+  padding-bottom: 10px;
+  display: flex;
+  flex-direction: column;
```

Comment:
`display: flex` and `flex-direction: column` don't seem to be doing anything for this element.